commons/src/test/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitControllerTests.scala

```diff
@@ -182,8 +181,8 @@ class PurePursuitControllerTests extends FunSuite {
     val target = Point(Feet(0), Feet(1))
     val position = hardware.turnPosition.mapToConstant(Point(Feet(1), Feet(1)))
 
-    // TODO: types not inferred correctly
-    val path: Stream[(Segment, Option[Segment])] = hardware.turnPosition.mapToConstant((Segment(origin, target), None))
+    val path: Stream[(Segment, Option[Segment])] =
```

Comment:
Ooh, this reminds me to jump on top of setting up appropriate type parameter variances for Streams. This will be solved once we make Streams have the type parameter be covariant.